lms/models/h_group.py

```diff
@@ -1,61 +1,24 @@
-from typing import NamedTuple
+import sqlalchemy as sa
 
-from lms.models._hashed_id import hashed_id
+from lms.db import BASE, TimestampedModelMixin
 
-MAX_GROUP_NAME_LENGTH = 25
 
+class HGroup(TimestampedModelMixin, BASE):
+    __tablename__ = "h_group"
+    __table_args__ = (sa.UniqueConstraint("name", "authority_provided_id"),)
```

Comment:
Is this right @seanh ? You mention something in slack identifying columns vs "updateable" ones. I'm using `authority_provided_id` as the only unique one and allowing name and type to change